### PR TITLE
Example for responsive table changed ( Fixing issue #5313) 

### PR DESCRIPTION
**Pull request by tanmaylaud**
This PR fixes issue #5313
I have added 50 columns so that the responsive behaviour can be clearly displayed ( even on larger windows )
Also, a sentence is added to the doc explaining the example.

### CODE REVIEW

**Comment by taion** [comment]
this seems like it makes the example quite unwieldy?

**Comment by taion** [comment]
that's a lot of columns...

**Comment by tanmaylaud** [comment]
> this seems like it makes the example quite unwieldy?

@taion  I have added many columns so that the responsive behaviour can be clearly shown. Even with browsers with large viewport. Regarding the example, I tried to keep it as simple as possible (without hurting the React Playground ). Could you suggest a better way to do it?

**Comment by taion** [comment]
We'd prefer the examples to match the upstream ones as much as possible: https://getbootstrap.com/docs/4.5/content/tables/#responsive-tables

**Comment by tanmaylaud** [comment]
> We'd prefer the examples to match the upstream ones as much as possible: https://getbootstrap.com/docs/4.5/content/tables/#responsive-tables

@taion Thanks for the reference. But, if you see, the upstream UI does not have a runnable playground like ours. So, we need to provide a runnable code. I had thought of directly putting the headers and cells. But then the code would become too lengthy even for 10 columns.

**Comment by taion** [comment]
yeah... the issue here is that the code becomes a little bit awkward/unidiomatic, and it's not quite "proper" in the sense that we don't have `key`s on the elements, &c.

can we just make the lists longer?

**Comment by taion** [comment]
like, if we get up to 12 columns, it should be enough to show the scrolling...

File: www/src/examples/Table/Responsive.js [hunk 1]
```diff
@@ -2,41 +2,37 @@
   <thead>
     <tr>
       <th>#</th>
-      <th>Table heading</th>
-      <th>Table heading</th>
-      <th>Table heading</th>
-      <th>Table heading</th>
-      <th>Table heading</th>
-      <th>Table heading</th>
+      {(function () {
```
Comment (mxschmitt) [inline www/src/examples/Table/Responsive.js:5]:
```suggestion
      {Array.from({length:5}).map(_ => <th>Table heading</th>)}
```
should work too for both cases.

**Comment by tanmaylaud** [comment]
> like, if we get up to 12 columns, it should be enough to show the scrolling...

Yeah..true. but the example code gets really long. Should I use the snippet suggested by @mxschmitt  instead ?

**Comment by taion** [comment]
in any event, it would be nice if this example set `key`s on the elements, at least

**Comment by tanmaylaud** [comment]
> in any event, it would be nice if this example set `key`s on the elements, at least

@taion I have changed the number of columns to 12, and added a key to each cell

File: www/src/examples/Table/Responsive.js (hunk 1 above)
Comment (tanmaylaud) [reply www/src/examples/Table/Responsive.js]: @mxschmitt  Thanks ! 👍

**Pull request merged**